Source/Urho3D/Scene/PrefabReference.cpp

```diff
@@ -230,15 +230,50 @@ void PrefabReference::RemoveInstance()
     instanceNode_ = nullptr;
 }
 
-void PrefabReference::InstantiatePrefab(const NodePrefab& nodePrefab)
+void PrefabReference::InstantiatePrefab(const NodePrefab& nodePrefab, PrefabInstanceFlags instanceFlags)
 {
     const auto flags = PrefabLoadFlag::KeepExistingComponents | PrefabLoadFlag::KeepExistingChildren
         | PrefabLoadFlag::LoadAsTemporary | PrefabLoadFlag::IgnoreRootAttributes;
     PrefabReaderFromMemory reader{nodePrefab};
     node_->Load(reader, flags);
+    if (instanceFlags != PrefabInstanceFlag::None) {
```

Comment:
Is all of this necessary? Can we just have 1 flag to copy parameters or not to copy? Is there a use case where users would prefer to set only subset of the attributes?

Reply:
There may be such cases, it's better to have control over what's copied and what's not.
And I think that it's better to write explicitly what you want to copy than spending time in debugger trying to figure out why that tag/rotation/... appeared in the node.